### PR TITLE
Improve error message for gas price oracle on V1

### PR DESCRIPTION
**Pull request by patitonar**
Related to https://github.com/poanetwork/tokenbridge/issues/84

New error message
gaspriceError

### CODE REVIEW

File: src/stores/GasPriceStore.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ class GasPriceStore {
       console.log(data)
       this.gasPrices = data;
     }).catch((e) => {
-      this.alertStore.pushError(e)
+      this.alertStore.pushError(`Gas price oracle is not available`)
```
Comment (akolotov) [inline src/stores/GasPriceStore.js]: So, what is our suggestion if the user see this message? Wait for a minute and try again or what?

Comment (patitonar) [reply src/stores/GasPriceStore.js]: The gas price oracle is fetched only when page is loaded, so message can be updated to also display suggestion to try to access the page again after a couple of minutes. Does it makes sense?

Comment (akolotov) [reply src/stores/GasPriceStore.js]: Yes, it should be more informative.

Comment (patitonar) [reply src/stores/GasPriceStore.js]: Thanks, fixed.

**Review by akolotov — APPROVED** [review]

**Pull request merged**